models/DataObject/ClassDefinition/Data/Classificationstore.php

```diff
@@ -1386,4 +1386,55 @@ public function denormalize($value, $params = [])
 
         return null;
     }
+
+
+    /**
+     * Creates getter code which is used for generation of php file for object classes using this data type
+     *
+     * @param DataObject\ClassDefinition|DataObject\Objectbrick\Definition|DataObject\Fieldcollection\Definition $class
+     *
+     * @return string
+     */
+    public function getGetterCode($class)
+    {
+        $key = $this->getName();
+
+        if ($class->getGenerateTypeDeclarations() && $this instanceof DataObject\ClassDefinition\Data\TypeDeclarationSupportInterface && $this->getReturnTypeDeclaration()) {
+            $typeDeclaration = ': ' . $this->getReturnTypeDeclaration();
+        } else {
+            $typeDeclaration = '';
```

Comment:
you can declare typeDeclaration = '', on top and remove the "else".

Reply:
Done 😉